src/test/resources/dlsfls/securityconfig_tlq.yml

```diff
@@ -0,0 +1,18 @@
+---
+_meta:
+  type: "config"
+  config_version: 2
+config:
+  dynamic:
+    do_not_fail_on_forbidden: true
+    authc:
+      authentication_domain_basic_internal:
+        http_enabled: true
+        transport_enabled: true
+        order: 0
+        http_authenticator:
+          challenge: false
+          type: "basic"
+        authentication_backend:
+          type: "intern"
+        description: "Migrated from v6"
```

Comment:
is this description really applicable here?

Reply:
True, I have removed it